README.md

```diff
@@ -8,9 +8,154 @@ OpenFisca is a versatile microsimulation free software. This repository contains
+
+- A [virtualenv](https://virtualenv.pypa.io/en/stable/) is a project specific environment created to suit the needs of the project you are working on.
+- A virtualenv manager, such as [pew](https://github.com/berdario/pew), lets you easily create, remove and toggle between several virtualenvs.
+
+To install pew, launch a terminal on your computer and follow these instructions:
+
+```sh
+pip install --upgrade pip
+pip install pew  # answer "Y" to the question about modifying your shell config file.
+```
+To set-up and run a virtualenv named **openfisca** running python2.7, run the command bellow.
+
+```sh
+pew new openfisca --python=python2.7
+```
+
+The virtualenv you just created will be automatically activated. This means you will operate in the virtualenv immediately.
+- Exit the virtualenv with **"exit"** (or Ctrl-D),
+- Re-enter with **"pew workon openfisca"**.
+
+### [EN] Minimal Installation (pip install)
+
+Follow this installation if you  wish to:
+- Run calculations on a large population ;
+- Create tax & benefits simulations ;
+- Write an extension to this legislation (e.g. city specific tax & benefits )
+- Serve this Country Package with the OpenFisca web API.
+
+For more advanced uses, head to the [Advanced Installation](#en-advanced-installation-git-clone)
+
+#### [EN] Install a Country Package with pip install
+
+Inside your virtualenv, check the prerequisites
+
+```sh
+python --version  # should print "Python 2.7.xx".
+#if not, make sure you pass the python version as an argument when creating your virtualenv
+```
+
+```sh
+pip --version  # should print at least 9.0.
+#if not, run "pip install --upgrade pip"
+```
+Install the OpenFisca Country Package.
+
+```sh
+#Example: Openfisca-France
+pip install openfisca-france
+```
+
+#### [EN] Next steps
+
+- To learn how to use Openfisca, follow our [tutorials](https://doc.openfisca.fr/getting-started.html) ;
+- To serve your country package with the [OpenFisca web API](#serve-your-country-package-with-the-openFisca-web-api) ;
+
+Depending on what you want to do with OpenFisca, you may want to install yet other packages in your virtualenv.
+- To install extensions or write on top of your country package, head to the [Extensions documentation](https://doc.openfisca.fr/contribute/extensions.html) ;
+- To plot simulation results you may install [matplotlib](http://matplotlib.org/) ;
+- To manage data you may install [pandas](http://pandas.pydata.org/).
+
+### [EN] Advanced installation (Git Clone)
+
+Follow this tutorial if you  wish to :
+- Create or change this Country Package's legislation
+- Contribute to the source code and to develop and/or fix part of it
+
+>***A note on OpenFisca-Core***: If you need to modify OpenFisca-Core source code, follow this [install tutorial](https://github.com/openfisca/openfisca-core#installation) section before completing the step below. By default just continue below.
```

Comment:
(e.g. you want to contribute to a Country Package agnostic feature) ? Je suis ouverte à d'autres suggestions :)